src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -2847,4 +2846,86 @@ public object ConvertBack(object value, Type targetType, object parameter, Syste
         }
     }
 
+    /// <summary>
+    /// Converter is used in WatchTree.xaml
+    /// It converts the boolean value of WatchViewModel.IsCollection to the background color of the each listnode label
+    /// </summary>
+    public class ListIndexBackgroundConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
+        {
+            if ((bool) value)
+            {
+                return "Transparent";
+            }
+            return "#aaaaaa";
+        }
+        public object ConvertBack(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
+        {
+            throw new NotImplementedException();
+        }
+    }
+
+    /// <summary>
+    /// Converter is used in WatchTree.xaml 
+    /// It converts the boolean value of WatchViewModel.IsCollection to determine the margin of the listnode textblock
+    /// </summary>
+
+    public class ListIndexMarginConverter : IValueConverter
```

Comment:
Is it possible to combine ListIndexMarginConverter with TopLevelLabelMarginConverter?